astropy/coordinates/earth.py

```diff
@@ -848,3 +864,84 @@ def _to_value(self, unit, equivalencies=[]):
             equivalencies = self._equivalencies
         new_array = self.unit.to(unit, array_view, equivalencies=equivalencies)
         return new_array.view(self.dtype).reshape(self.shape)
+
+
+class BaseGeodeticRepresentation(BaseRepresentation):
+    """
+    Representation of points in 3D geodetic coordinates.
+
+    Parameters
+    ----------
+    lon, lat : `~astropy.units.Quantity`
```

Comment:
The classes are `Longitude` and `Latitude` - might as well be precise...

Reply:
Tried this trick https://github.com/astropy/astropy/issues/11063#issuecomment-732212333 let's see if it works!